package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "apollo-boost": "^0.4.7",
+    "gh-pages": "^3.0.0",
```

Comment:
do you need this in production? Shouldn't it be a dev-dependency? By the way, why do you have mobx? And you also have redux?

Reply:
I've removed them.